pypln/web/settings.py

```diff
@@ -18,98 +18,96 @@
 # along with PyPLN.  If not, see <http://www.gnu.org/licenses/>.
 import os
 
+from decouple import config, Csv
+from dj_database_url import parse as db_url
+
+try:
+    import urlparse
+except ImportError:
+    import urllib.parse as urlparse
+
+
+def parse_url(url):
+    urlparse.uses_netloc.append('mongodb')
+    url = urlparse.urlparse(url)
+
+    path = url.path[1:]
+    path = path.split('?', 2)[0]
+
+    return {
+        'database': path or '',
+        'port': url.port or '',
+        'host': url.p or '',
+    }
+
+    return config
+
 PROJECT_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__),
     os.path.pardir))
 
-DEBUG = True
+DEBUG = config('DEBUG', default=False, cast=Csv())
 TEMPLATE_DEBUG = DEBUG
 
+EMAIL_BACKEND = config('EMAIL_BACKEND', default='django.core.mail.backends.console.EmailBackend')
+
+if EMAIL_BACKEND != 'django.core.mail.backends.console.EmailBackend':
+    EMAIL_HOST, EMAIL_PORT, EMAIL_HOST_USER, EMAIL_HOST_PASSWORD = config('EMAIL_CONFIG', default='example_host,25,username,password', cast=Csv())
+
```

Comment:
We used to set `EMAIL_USE_TLS` to `True` in `settings.production`. This may not be necessary anymore, but we should check.